src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -436,6 +436,12 @@
         </xsd:restriction>
     </xsd:simpleType>
 
+    <xsd:complexType name="default_middleware_elem">
```

Comment:
What about keeping `default_middleware` for this one?

Reply:
For BC with XML configuration.

Reply:
This should not be necessary since BC is already broken with the current renaming.
AFAIK we cannot deprecate types in XSD, so should we bother?

Reply:
I think it's OK because it's just a name for seperate config.

https://symfony.com/doc/current/messenger/multiple_buses.html